### PR TITLE
Products Proposal modal update

### PR DESCRIPTION
**Pull request by riadmen**
https://github.com/metasfresh/metasfresh/issues/16430

### CODE REVIEW

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5705850_sys_gh16430_Missing_Base_PLV_Msg.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+-- Value: WEBUI_Missing_Base_PriceList_Version
+-- 2023-10-10T18:06:53.914Z
+UPDATE AD_Message_Trl SET IsTranslated='Y', MsgText='Pricelist has no Base Pricelist',Updated=TO_TIMESTAMP('2023-10-10 19:06:53','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='en_US' AND AD_Message_ID=545352
+;
+
+-- Value: WEBUI_Missing_Base_PriceList_Version
+-- 2023-10-10T18:06:59.408Z
+UPDATE AD_Message_Trl SET MsgText='Pricelist has no Base Pricelist',Updated=TO_TIMESTAMP('2023-10-10 19:06:59','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='fr_CH' AND AD_Message_ID=545352
+;
```
Comment (mrie04) [inline backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5705850_sys_gh16430_Missing_Base_PLV_Msg.sql:33]: This one isn't translated. Pls flag `de_DE` and `de_CH` instead

File: backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/order/products_proposal/process/WEBUI_ProductsProposal_ShowProductsToAddFromBasePriceList.java [hunk 2]
```diff
@@ -38,7 +47,10 @@ protected ProcessPreconditionsResolution checkPreconditionsApplicable()
 	{
 		if (!getView().getBasePriceListVersionId().isPresent())
 		{
-			return ProcessPreconditionsResolution.rejectWithInternalReason("no base price list set");
+			final BPartnerId bPartnerId = getView().getBpartnerId().orElse(null);
+			final I_C_BPartner bPartner = partnerDAO.getById(bPartnerId);
+			final ITranslatableString msg = msgBL.getTranslatableMsgText(MSG_MissingBasePriceListVersion, bPartner.getName());
```
Comment (mrie04) [inline backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/order/products_proposal/process/WEBUI_ProductsProposal_ShowProductsToAddFromBasePriceList.java:52]: Parameter can be removed.

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5705850_sys_gh16430_Missing_Base_PLV_Msg.sql [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+-- Value: WEBUI_Missing_Base_PriceList_Version
+-- 2023-10-10T18:06:03.702Z
+UPDATE AD_Message SET MsgText='Preisliste hat keine Basis Preisliste',Updated=TO_TIMESTAMP('2023-10-10 19:06:03','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Message_ID=545352
+;
+
+-- Value: WEBUI_Missing_Base_PriceList_Version
+-- 2023-10-10T18:06:47.440Z
+UPDATE AD_Message_Trl SET MsgText='Pricelist has no Base Pricelist',Updated=TO_TIMESTAMP('2023-10-10 19:06:47','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Language='de_CH' AND AD_Message_ID=545352
```
Comment (mrie04) [inline backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5705850_sys_gh16430_Missing_Base_PLV_Msg.sql:22]: Should be removed. Correct german translation is added on insert.

**Review by mrie04 — CHANGES_REQUESTED** [review]
Pls check my comments

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5705850_sys_gh16430_Missing_Base_PLV_Msg.sql (hunk 1 above)
Comment (riadmen) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5705850_sys_gh16430_Missing_Base_PLV_Msg.sql:33]: done

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5705850_sys_gh16430_Missing_Base_PLV_Msg.sql (hunk 3 above)
Comment (riadmen) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5705850_sys_gh16430_Missing_Base_PLV_Msg.sql]: done

File: backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/order/products_proposal/process/WEBUI_ProductsProposal_ShowProductsToAddFromBasePriceList.java (hunk 2 above)
Comment (riadmen) [reply backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/order/products_proposal/process/WEBUI_ProductsProposal_ShowProductsToAddFromBasePriceList.java]: done

**Review by mrie04 — APPROVED** [review]
lgtm

**Comment by metas-ts** [comment]
@mergifyio copy intensive_care_hotfix

**Pull request merged**